qiskit/visualization/state_visualization.py

```diff
@@ -1136,19 +1136,14 @@ def state_to_latex(
     return prefix + latex_str + suffix
 
 
-def _round_if_close(data):
-    """Round real and imaginary parts of complex number of close to zero"""
-    data = np.real_if_close(data)
-    data = -1j * np.real_if_close(data * 1j)
-    return data
-
-
-def num_to_latex_ket(raw_value: complex, first_term: bool) -> Optional[str]:
+def num_to_latex_ket(raw_value: complex, first_term: bool, max_size: int) -> Optional[str]:
```

Comment:
_Technically_ this is a public function, so we should supply a default argument for `max_size` to avoid breaking existing uses.

Reply:
set to 0 in 5c86cdcd7201d0ea7085dd4b74604f587140d84e